frappe/public/js/frappe/form/footer/attachments.js

```diff
@@ -13,6 +13,9 @@ frappe.ui.form.Attachments = Class.extend({
 		this.parent.find(".add-attachment").click(function() {
 			me.new_attachment();
 		});
+		this.parent.find(".add-gsuite").click(function() {
```

Comment:
As this isn't an upload (the template is already in google drive), we should change the popup title "upload attachment" to something like "Add Attachment" to be more service agnostic.
It seems to me that this is a major change js files (as far i found, the click on attach button is defined in frappe.upload.make. Do you think I should only add there the code for Gsuite or you what to make any kind of abstraction?